src/main/resources/application.yml

```diff
@@ -1,23 +1,28 @@
 spring:
   datasource:
-    url: jdbc:postgresql://localhost:5432/find_hai
+#    url: jdbc:postgresql://localhost:5432/find_hai
+    url: jdbc:postgresql://172.105.109.16:5433/find_hai
     username: postgres
-    password: password
-
+    password: findhai
```

Comment:
@nelsonkimaiga, why do we have passwords pasted one these publicly accesible configs?